ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -161,6 +161,23 @@ l3leaf:
           mgmt_ip: 192.168.200.118/24
           mac_address: '0c:1d:c0:1d:62:01'
           uplink_switch_interfaces: [ Ethernet9, Ethernet9, Ethernet9, Ethernet9 ]
+    DC1_CL1:
+      mlag: true
+      evpn_services_l2_only: true
+      filter:
+        tenants: [ Tenant_A, Tenant_B, Tenant_C ]
+        tags: [ opzone, web, app, db, vmotion, nfs, wan ]
+      nodes:
+        DC1-CL1A:
+          id: 10
+          bgp_as: 65108
+          mgmt_ip: 192.168.200.119/24
+          uplink_switch_interfaces: [ Ethernet10, Ethernet10, Ethernet10, Ethernet10 ]
```

Comment:
these uplink interfaces were already used in some other test scenario (if you check config and doc updates for spine1, you can see it changing the ip and description instead of adding new interfaces). I suggest to use Ethernet12 and 13

Reply:
okay, i'll update the intf to Et12, and Et13

Reply:
et12 was being used by a other leaf too, thus went ahead with Et14-15